conf/rest/9.7.0/nfs_clients.yaml

```diff
@@ -1,3 +1,5 @@
+# This template is disabled by default because It may slow down data collection
```

Comment:
minor but `It` should be `it`

This template is disabled by default because it may slow down data collection